src/main/java/Bundle_fr.properties

```diff
@@ -8,8 +8,8 @@ datasets=Ensembles de donn
 newDataset=Nouvel ensemble de donn�es
```

Comment:
When will this entire Bundle_fr.properties file be removed? It's the plan for installations who want French support to get the Bundle_fr.properties file from https://github.com/GlobalDataverseCommunityConsortium/dataverse-language-packs ?

Reply:
@pdurbin - Yes, I would like to create a separate issue to remove all french language files and Bundle_zh_CN.properties from codebase.  Also to move all the English property files into a specific folder.  Eg:  under src/main/java/english/*.properties , so this folder can be synchronized automatically (through Jenkins setup) to the English folder in dataverse-language-packs repository.  Later, translators can work for other languages.  Please add your comments.  Thanks